source/reference/limitations.txt

```diff
@@ -0,0 +1,73 @@
+.. _c2c-limitations:
+
+===========
+Limitations
+===========
+
```

Comment:
Can we add a small note below here saying: "Some of the limitations may be removed from this list in the Generally Available version"

Reply:
+1